tests/test_evaluation_harness/configs/html_content_element_exact_match.json

```diff
@@ -1,8 +1,8 @@
 {
-    "sites": ["shopping"],
+    "sites": ["gitlab"],
```

Comment:
Can we revert this back to shopping? Same for the tests below. Reason is that gitlab isn't always set in vwa, and it doesn't seem to be necessary for these unit tests (they don't invoke the simulated environments).